readthedocs/templates/projects/project_users.html

```diff
@@ -24,14 +24,14 @@ <h3>{% trans "Existing Maintainers" %}</h3>
     {% endfor %}
     </ul>
 
-  {% if users|length > 1 %}
+  {% if not is_last_user %}
   <h3>{% trans "Remove Maintainer" %}</h3>
     <p>
         <form method="post" action="{% url "projects_users_delete" project.slug %}">
             {% csrf_token %}
           <select id="id_user" name="username">
           {% for user in users %}
-            {% if user != request.user %}
+            {% if not is_last_user or user != request.user %}
```

Comment:
This logic doesn't make sense to me. Seems we're evaluating the variable every loop? Isn't that accounted for by the `{% if not is_last_user %}` block above?

Reply:
the value is evaluated once, but yeah, we don't show the selector at all if you are the only one, so we can just remove this conditional